digitalpublicgoods/kolibri.json

```diff
@@ -0,0 +1,293 @@
+{
+  "name": "Kolibri",
+  "clearOwnership": {
+    "isOwnershipExplicit": "Yes",
+    "copyrightURL": "https://drive.google.com/file/d/1gquJaNTtRiAsu85b2NNDumqm1cPsK_iK/view?usp=sharing"
```

Comment:
@nathanbaleeta are we OK with this links to Google Drive? Can we suggest to put those PDF files in the repository and use the repository permalink?

Reply:
@iperdomo Links to Google drive are okay considering they are not password protected.